spaces/domains.json

```
@@ -58,7 +58,8 @@
   "signal.idle.finance": "idlefinance.eth",
   "gov.flush.lol": "flushlol.eth",
   "fund.emptyset.finance": "esd.eth",
-  "mtrg.meter.io": "meter-mainnet.eth",
+   "mtrg.meter.io": "meter-mainnet.eth",
```

Beware of side effects.
```suggestion
  "mtrg.meter.io": "meter-mainnet.eth",
```